networking/ingress-sharding.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="ingress-sharding"]
+= Ingress sharding in {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: ingress-sharding
+
+toc::[]
+
+In {product-title}, an Ingress Controller can serve all routes, or it can serve a subset of routes. By default, the Ingress Controller will serve any route created in any namespace in the cluster. Additional Ingress Controllers can be added to your cluster to optimize routing by creating _shards_, which are subsets of routes based on selected characteristics. To mark a route as a member of a shard, use labels in the route or namespace `metadata` field. The Ingress Controller uses _selectors_, also known as a _selection expression_, to select a subset of routes from the entire pool of routes to serve.
+
+Ingress sharding, or router sharding, is useful in cases where you want to load balance incoming traffic across multiple Ingress Controllers, when you want to isolate traffic to be routed to a specific Ingress Controller, or for a variety of other reasons described in the next section.
```

Comment:
Is ingress sharding a.k.a. router sharding? I'd introduce that in the leading paragraph if so and leave this one without "or router sharding."

Reply:
Could also format these cases as an unordered list.

Reply:
yeah ingress sharding = router sharding, just updated 4.x terminology. I think i'll just omit the "router sharding" part, I added it for familiarity for customers coming from 3.10 where router sharding was used, but it's clear enough without it.